aglio-olio.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Aglio Olio</title>
+  </head>
+  <body>
+    <div>
+      <h1>Aglio Olio</h1>
+      <img
+        src="https://www.allrecipes.com/thmb/C6X4A92-CvGVJizfjkoUKumF_-g=/750x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/8962968_original-Tammy-2000-ce9b060431df4d3cb3b73b760b970bcc.jpg"
+        alt="Aglio Olio"
+        width="400"
+      />
+    </div>
+    <div>
```

Comment:
nice use of div here. Since you want to separate a section, can also use the `<section>` tag for more semantically correct html